### PR TITLE
Feature/automation cleanup sql query

### PR DESCRIPTION
**Pull request by sinhaamit1980**

### CODE REVIEW

File: design/census-import/yarn.lock [hunk 1]
```diff
@@ -259,9 +259,9 @@ kuler@1.0.x:
     colornames "^1.1.1"
 
 lodash@^4.17.11:
-  version "4.17.11"
-  resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.11.tgz#b39ea6229ef607ecd89e2c8df12536891cac9b8d"
-  integrity sha512-cQKh8igo5QUhZ7lg38DYWAxMvjSAKG0A8wGSVimP07SIUEK2UO+arSRKbRZWtelMtN5V0Hkwh5ryOto/SshYIg==
+  version "4.17.15"
+  resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.15.tgz#b447f6670a0455bbfeedd11392eff330ea097548"
+  integrity sha512-8xOcRHvCjnocdS5cpwXQXVzmmh5e5+saE2QGoeQmbKmRS6J3VQppPOIt0MnmE+4xlZoumy0GPG0D0MVIQbNA1A==
```
Comment (activemq) [inline design/census-import/yarn.lock]: @sinhaamit1980 why are you getting a conflict on this? Dont think making this change is correct

Comment (sinhaamit1980) [reply design/census-import/yarn.lock]: I think there was a dependancy ticket for census import which got merged before mine and due to conflict, it got replaced with mine. I replaced this with from master branch

File: test/pupil-hpa/features/support/sql_db_helper.rb [hunk 2]
```diff
@@ -90,24 +90,23 @@ def self.find_pupil_from_school(first_name, school_id)
...
+  #   sql = "UPDATE [mtc_admin].[pupil] set pin=#{flag}, pinExpiresAt= '#{new_time}' WHERE foreName='#{forename}' AND lastName='#{lastname}' AND school_id='#{school_id}'"
+  #   result = SQL_CLIENT.execute(sql)
+  #   result.do
+  # end
+
+  # def self.set_pupil_pin_expiry(forename,lastname,school_id,new_time)
+  #   sql = "UPDATE [mtc_admin].[pupil] set pinExpiresAt='#{new_time}' WHERE foreName='#{forename}' AND lastName='#{lastname}' AND school_id='#{school_id}'"
+  #   result = SQL_CLIENT.execute(sql)
+  #   result.do
+  # end
```
Comment (activemq) [inline test/pupil-hpa/features/support/sql_db_helper.rb]: @sinhaamit1980 Why leave these commented, just delete them?

File: test/admin-hpa/features/support/sql_db_helper.rb [hunk 3]
```diff
@@ -46,17 +46,17 @@ def self.find_teacher(name)
...
+  #   sql = "UPDATE [mtc_admin].[checkPin] set pinExpiresAt='2018-12-12 23:00:59.999 +00:00'"
+  #   result = SQL_CLIENT.execute(sql)
+  #   result.do
+  # end
+
+  # def self.set_pupil_pin_expiry(forename, lastname, school_id, new_time)
+  #   sql = "UPDATE [mtc_admin].[pupil] set pinExpiresAt='#{new_time}' WHERE foreName='#{forename}' AND lastName='#{lastname}' AND school_id='#{school_id}'"
+  #   result = SQL_CLIENT.execute(sql)
+  #   result.do
+  # end
```
Comment (activemq) [inline test/admin-hpa/features/support/sql_db_helper.rb]: @sinhaamit1980 Delete commented code if its doing nothing

File: test/pupil-hpa/features/support/sql_db_helper.rb (hunk 2 above)
Comment (sinhaamit1980) [reply test/pupil-hpa/features/support/sql_db_helper.rb]: Done

File: test/admin-hpa/features/support/sql_db_helper.rb (hunk 3 above)
Comment (sinhaamit1980) [reply test/admin-hpa/features/support/sql_db_helper.rb]: Done

**Pull request merged**